test/e2e/framework/test_context.go

```diff
@@ -320,6 +320,7 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 
 	flags.BoolVar(&TestContext.ListImages, "list-images", false, "If true, will show list of images used for runnning tests.")
 	flags.BoolVar(&TestContext.ListConformanceTests, "list-conformance-tests", false, "If true, will show list of conformance tests.")
+	flags.BoolVar(&TestContext.PrepullImages, "prepull-images", true, "If true, prepull images so image pull failures do not cause test failures.")
```

Comment:
true by default?

Reply:
Hm, good point. I've moved this from ``e2e_node_suite_test.go`` and I didn't check the value. I'd keep it as ``false`` for E2E, and keep it ``true`` for Node E2E.

Reply:
let's ask @spiffxp  , but I'd keep this false at least a few releases to avoid "surprises" on downstream CIs that may expect the tests to start quickly

Reply:
Done